setup.cfg

```diff
@@ -48,26 +48,28 @@ asdf_extensions =
 test =
     pytest-astropy>=0.8
     pytest-xdist
+    coverage
+test_all =
     objgraph
     ipython
-    coverage
     skyfield>=1.20
     sgp4>=2.3
-all =
+rec =
```

Comment:
"rec" is too cryptic for me.
```suggestion
recommended =
```

Reply:
Yes, quite happy for "recommended" - still OK to use `rec` in `tox.ini`?

Reply:
Yeah, I don't think typical user would care about what we name things in `tox.ini`.